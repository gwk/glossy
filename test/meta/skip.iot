{
  'cmd': '$PROJ/iotest.py -no-times test-meta/skip.iot',
  'code': 1,
  'links' : {'test-meta'},
}
