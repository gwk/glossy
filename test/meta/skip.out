test-meta/skip                                                   SKIPPED.

################################################################
TESTS FOUND: 1; SKIPPED: 1; PASSED: 0.
